index.html

```diff
@@ -1030,25 +1030,23 @@ <h2>Query</h2>
         <h2>Fragment</h2>
 
         <p>
-A <a>DID fragment</a> is used as method-independent reference into the <a>DID
-document</a> to identify a component of the document (for example, a unique
-<a>public key description</a> or <a>service endpoint</a>). <a>DID fragment</a>
-syntax and semantics are identical to a generic URI fragment and  MUST conform
-to <a data-cite="RFC3986#section-3.5">RFC&nbsp;3986, section 3.5</a>. To
-dereference a <a>DID fragment</a>, the complete <a>DID URL</a> including the
+A <a>DID fragment</a> is used as method-independent reference into a <a>DID Document</a> or secondary resource.
```

Comment:
Probably not entirely clear what "secondary resource" means... have we looked at the URI spec yet to see if it has some language for "not in this document, but possibly another one encoded in a URI parameter?"

Reply:
I have not, but I would welcome a citation to that spec that might be helpful to provide additional detail. 

I think "secondary resource" by itself is good, but a link to more detail would be helpful.

Reply:
What about "external resource"?